tests/HOMESCREEN/test_26780.py

```diff
@@ -68,15 +68,11 @@ def test_run(self):
         simon.tap()
         time.sleep(5)
         self.marionette.switch_to_frame()
-        self.UTILS.element.waitForElements(DOM.EME.footer_navigation_bar, "Footer navigation bar", timeout=30)
-        bar = self.UTILS.element.getElement(DOM.EME.footer_navigation_bar, "Footer navigation bar")
-        bar.tap()
+        menu = self.UTILS.element.getElement(DOM.EME.launched_menu_button, "Menu button")
+        menu.tap()
 
-        self.UTILS.element.waitForElements(DOM.EME.launched_button_back, "Button bar - back button")
-        self.UTILS.element.waitForElements(DOM.EME.launched_button_forward, "Button bar - forward button")
-        self.UTILS.element.waitForElements(DOM.EME.launched_button_reload, "Button bar - reload button")
-        self.UTILS.element.waitForElements(DOM.EME.launched_button_bookmark, "Button bar - bookmark button")
-        self.UTILS.element.waitForElements(DOM.EME.launched_button_close, "Button bar - close button")
-
-        x = self.UTILS.debug.screenShotOnErr()
-        self.UTILS.reporting.logResult("info", "Screenshot of the button bar:", x)
+        self.UTILS.element.waitForElements(DOM.EME.launched_button_new_window, "New window button")
```

Comment:
The test description asks to check that every expected button is in place... I think they are required.